vector/src/main/java/im/vector/app/features/home/HomeActivity.kt

```diff
@@ -218,7 +218,7 @@ class HomeActivity :
                 fcmHelper.ensureFcmTokenIsRetrieved(
```

Comment:
I think here we should only have the `activity` as parameter for that method. `PusherManager` can be injected in `GoogleFcmHelper` and the check to add a pusher or not could be moved to the `GoogleFcmHelper` as well. What do you think?

Reply:
Adding the check to `GoogleFcmHelper` was my first idea but this check involves the session and pushers service. I think we don't need to add these dependencies to `GoogleFcmHelper`. Can we also revisit this later as although I agree that we should try to reduce the number of parameters we pass, I don't want to change too much at this late stage

Reply:
Okay, no problem let's update it later.